homeworks/natalia.kokovina_NataliKokovina/homework_3/function.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-shadow */
+
+// 1.1 Exponentiation
+
+function Exponentiation(testNumber) {
+    console.log(testNumber ** 2);
+}
+Exponentiation(5);
+Exponentiation(15);
+Exponentiation(53);
+
+// 1.2 Make an array
+
+function array(max, step) {
+    const arr = [];
+    for (let i = 0; i <= max; i += step) {
+        arr.push(i);
+    }
+    console.log(arr);
+    return (arr);
+}
+array(50, 5);
+array(32, 4);
+
+//  1.3 Exponentiation elements of arr
+
+function Expon(numSet) {
```

Comment:
also, no capitalized function naming unless it's a constructor

Reply:
fix